### DISCUSSION TITLE
[CFP] DASFAA 2023 - April 17-20 , 2023 | Tianjin, China

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#171] [3️⃣ 聊学术]
DASFAA 一直是 CCF B 类论文发表的不错选择，和数据库、数据应用系统、数据科学等内容也非常相关。值得针对性的考虑与投稿~

http://www.tjudb.cn/dasfaa2023/important_dates

<html>
<body>
<h1 align="left" style="font-size: 1.357em; margin: 1em 0px 0.5em; color: rgb(0, 0, 0); font-weight: inherit; font-family: Georgia, &quot;Times New Roman&quot;, Times, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><b style="font-weight: bolder;"><span style="font-family: &quot;Century Gothic;, serif&quot;;">Conference Dates</span></b></h1>

Main conference:: | April 17-20, 2023
-- | --

</body>
</html>

<html>
<body>
<h1 align="left" style="font-size: 1.357em; margin: 1em 0px 0.5em; color: rgb(0, 0, 0); font-weight: inherit; font-family: Georgia, &quot;Times New Roman&quot;, Times, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><b style="font-weight: bolder;"><span style="font-family: &quot;Century Gothic;, serif&quot;;">Research Papers</span></b></h1>

Abstract submission deadline: | October 28, 2022
-- | --
Paper submission deadline: | November 04, 2022
Acceptance notification: | January 09, 2023
Camera ready due: | January 23, 2023

</body>
</html>

<html>
<body>
<h1 align="left" style="font-size: 1.357em; margin: 1em 0px 0.5em; color: rgb(0, 0, 0); font-weight: inherit; font-family: Georgia, &quot;Times New Roman&quot;, Times, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><b style="font-weight: bolder;"><span style="font-family: &quot;Century Gothic;, serif&quot;;">Industry Papers</span></b></h1>

Paper submission deadline: | November 18, 2022
-- | --
Acceptance notification: | December 28, 2022
Camera ready due: | January 27, 2023

</body>
</html>

<html>
<body>
<h1 align="left" style="font-size: 1.357em; margin: 1em 0px 0.5em; color: rgb(0, 0, 0); font-weight: inherit; font-family: Georgia, &quot;Times New Roman&quot;, Times, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><b style="font-weight: bolder;"><span style="font-family: &quot;Century Gothic;, serif&quot;;">Demo Papers</span></b></h1>

Paper submission deadline: | November 28, 2022
-- | --
Acceptance notification: | January 20, 2023
Camera ready due: | January 27, 2023

</body>
</html>

<html>
<body>
<h1 align="left" style="font-size: 1.357em; margin: 1em 0px 0.5em; color: rgb(0, 0, 0); font-weight: inherit; font-family: Georgia, &quot;Times New Roman&quot;, Times, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><b style="font-weight: bolder;"><span style="font-family: &quot;Century Gothic;, serif&quot;;">Ph.D. Consortium Papers</span></b></h1>

Paper submission deadline: | November 28, 2022
-- | --
Acceptance notification: | January 04, 2023
Camera ready due: | January 27, 2023

</body>
</html>

**Comment by bifenglin** [comment]
Research Papers貌似28号就需要提交Abstract。。。

**Reply by will-ww** [reply]
好好努力，争取明年赶上。不过 Ph.D. Consortium 可以考虑，和开题联动一下～
